coverage/pytracer.py

```diff
@@ -15,6 +15,9 @@
     YIELD_VALUE = chr(YIELD_VALUE)
 
 
+HAS_F_TRACE_LINES = sys.version_info >= (3, 7)
+
```

Comment:
I'd rather add a constant to PYBEHAVIOR in env.py

Reply:
Would it require to get a frame object first?  (i.e. `sys._getframe()`) to inspect it?

Reply:
Sure, though that should be easy.  Though come to think of it, why not just do the hasattr where you try to set the attribute?

Reply:
My idea was to have as less overhead as possible in the tracing function itself.